examples/getting_started/pt/src/cifar10_fl.py

```diff
@@ -122,6 +122,7 @@ def evaluate(input_weights):
 
         # (6) evaluate on received model for model selection
         accuracy = evaluate(input_model.params)
+        summary_writer.add_scalar(tag="global_model_accuracy", scalar=accuracy, global_step=input_model.current_round)
```

Comment:
if you think this is necessary, could you move this change to another PR?
So this PR only relates to TF changes, thanks

Reply:
@YuanTingHsieh,

our changes just involve nvflare/app_opt/tf/scaffold.py and some of the files within examples/getting_started/tf.
Besides that, no changes where made by us. We forked https://github.com/holgerroth/NVFlare/tree/tf_algos and other changes stated above should be related to this work.
We used this branch because of the the newly introduced workflow and a working implementation of FedOpt.